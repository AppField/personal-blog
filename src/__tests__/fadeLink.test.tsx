import React from 'react';
import renderer from 'react-test-renderer';
import InternalProvider from 'gatsby-plugin-transition-link/context/InternalProvider';

import FadeLink from '../components/fadeLink';

describe('<FadeLink/>', () => {
  it('renders correctly', () => {
    global.window.matchMedia = jest.fn().mockImplementation(() => true);

    const tree = renderer
      .create(
        <InternalProvider>
          <FadeLink to="/blog" />
        </InternalProvider>
      )
      .toJSON();

    expect(tree).toMatchSnapshot();
  });
});
