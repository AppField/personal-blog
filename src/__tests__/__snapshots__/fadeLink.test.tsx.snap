// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<FadeLink/> renders correctly 1`] = `
<a
  href="/blog"
  onClick={[Function]}
  onMouseEnter={[Function]}
/>
`;
